src/mmw/js/src/modeling/models.js

```diff
@@ -955,11 +961,16 @@ var ScenariosCollection = Backbone.Collection.extend({
         return this.setActiveScenario(this.get({ cid: cid }));
     },
 
-    createNewScenario: function(aoi_census) {
-        var scenario = new ScenarioModel({
-            name: this.makeNewScenarioName('New Scenario'),
-            aoi_census: aoi_census
-        });
+    createNewScenario: function() {
+        var current = this.findWhere({ 'is_current_conditions': true }),
```

Comment:
Consider renaming `current` to `currentConditions`, as `current` can imply the currently selected scenario.